### ISSUE TITLE
[Bug] The HTTP API Creates an unnecssary / unwanted Connect Address due to HTTPListener Constructor

### ISSUE THREAD

**Issue by 3lpsy** [cobbr/Covenant#151]
The ListenerApiController creates an extra Connect Address due to the HTTPListener constructor when creating a new HTTP listener. 

API Call: 
```
         // When the HTTPListener is initialized from the request data
         // It has a default connect address of 127.0.0.1 (or whatever is resolved, typically private ip)
         // Any connect address submitted to the API are appended to the list ["127.0.0.1"]
        [HttpPost("http", Name = "CreateHttpListener")]
        public async Task<ActionResult<HttpListener>> CreateHttpListener([FromBody] HttpListener listener)
        {
            try
            {
                return await _context.CreateHttpListener(_userManager, _configuration, listener, _ListenerCancellationTokens, _eventhub);
            }
            catch (ControllerNotFoundException e)
            {
                return NotFound(e.Message);
            }
            catch (ControllerBadRequestException e)
            {
                return BadRequest(e.Message);
            }
        }
```

Constructor: 
```
     public HttpListener()
    {
      this.Description = "Listens on HTTP protocol.";
      try {
         this.ConnectAddresses = new List<string> {
                    Dns.GetHostAddresses(Dns.GetHostName())
                        .FirstOrDefault(A => A.AddressFamily == System.Net.Sockets.AddressFamily.InterNetwork)
                         .ToString()
                };
...
```

I tried removing this feature from the constructor but it breaks the create listener View as it expects the list to not be empty.

If the constructor feature was added to create default data for the View, I suggest creating a different constructor. If this is the case, let me know and I can PR a potential solution.

**Comment by cobbr** [comment]
@3lpsy  I think this should be fixed in the new version 79c45e7

Can you verify this problem is fixed for you?

**Comment by cobbr** [comment]
Closing for now, feel free to reopen if you continue to have problems.
